### PR TITLE
Add role to set up presentation clients.

### PR DESCRIPTION
**Pull request by nickygerritsen**

### CODE REVIEW

File: provision-contest/ansible/presclient.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    GRAPHICAL: true
+  become: true
+  handlers:
+    - include: handlers.yml
+  roles:
+    - role: base_packages
+      tags: base_packages
+    - role: icpc_fixes
+      tags: icpc_fixes
+      when: ICPC_IMAGE
```
Comment (vmcj) [inline provision-contest/ansible/presclient.yml:17]:
```suggestion
```

File: provision-contest/ansible/roles/presclient/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+# These tasks configure the presentation clients
+
+- name: Get the presentation client release
+  uri:
+    url: https://api.github.com/repos/icpctools/icpctools/releases?per_page=1
+    method: GET
+    return_content: true
+    status_code: 200
+    body_format: json
+  register: latest_pres_cliejt_release_array
```
Comment (vmcj) [inline provision-contest/ansible/roles/presclient/tasks/main.yml:11]: I like how this works in every language.

File: provision-contest/ansible/roles/presclient/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    presentation_account: "{{ CDS_ACCOUNTS | selectattr('username', '==', 'presentation') }}"
+
+- name: Add presentation client script
+  template:
+    src: presclient.j2
+    dest: /usr/local/bin/presclient
+    owner: domjudge
+    group: domjudge
+    mode: 0755
+
```
Comment (vmcj) [inline provision-contest/ansible/roles/presclient/tasks/main.yml:49]:
```suggestion
```

**Review by vmcj — APPROVED** [review]
Couple nits, but seems to be fine.

File: provision-contest/ansible/presclient.yml (hunk 1 above)
Comment (nickygerritsen) [reply provision-contest/ansible/presclient.yml:17]: I prefer to keep this for if we ever do need this.

**Pull request merged**